rdmo/core/models.py

```diff
@@ -44,3 +45,21 @@ def trans(self, field):
                         if r is not None:
                             return r
         return ''
+
+class ElementSitePermissionsMixin(models.Model):
```

Comment:
As discussed, I would copy-paste the elements in the seperate models. Not super elegeant, but less error prone. Otherwise we need to find a way to adjust the help text, without messing up the translations.

Reply:
yes, the `sites` field is only relevant for the end-user and the catalog (and not for the other models).
Then the `editor` field goes like normal in the separate models!
We also would like to have view permissions per object , or does that go via a queryset filter for editor?

Reply:
I would do that with filtering, I think sometimes people need to see elements they can not change.

Reply:
removed this Mixin model and reverted to original